### DISCUSSION TITLE
Allow the Not Found Message to Exists for Unsearchable Selects and Multiselects

### DISCUSSION THREAD

**Discussion by AustinWildgrube** [mantinedev/mantine#6474] [Feature requests]
Can we allow the `notFoundMessage` prop to show even when the select is not searchable?

I often get questions from users about the select and/or multiselect not working when there's no data available. For example, on my products page, users can't add duplicate products to their cart. To make this happen, I filter the dropdowns to remove already existing products, which sometimes leaves the multiselect with no data. Users then try to click the it and get confused when the dropdown doesn't open. Right now, there's the `notFoundMessage` prop, but it's only available when the select is searchable.

I'm willing to create a PR to address this issue, but I'd like to hear opinions before I spend time on it. Should we allow the not found message to show even when the select isn't searchable? Should I add it as a separate prop (I don't think so)? Or is there a better solution?

**Comment by rtivital** [comment]
I'm fine with these changes, you are welcome to submit a PR
